tests/p2p/bootstrap_test/test_restart_bootstrap.py

```diff
@@ -0,0 +1,64 @@
+from pytest_testconfig import config as testconfig
+import time
+
+from tests.conftest import DeploymentInfo
+from tests import pod
+from tests.test_bs import setup_bootstrap_in_namespace, add_curl, setup_bootstrap
+from tests.misc import CoreV1ApiClient
+from tests.utils import get_spec_file_path
+
+
+BOOTSTRAP_LABEL = 'bootstrap'
+
+
+# reboot bootstrap
+# add a new stateful bootstrap
+# kill bootstrap
+# create a new bootstrap with the former key
+# validate state is the same as the old bootstrap node
+def test_reboot_bootstrap(init_session):
+    print("test_reboot_bootstrap")
+    # test params
+    sleep_time = 5
+    session_id = init_session
+    bootstrap_group_id = 'bootstrap_key'
```

Comment:
This assignment is not needed. Just set it in line no. 26

Reply:
it's more readable this way otherwise you have to know what value init_session returns